blogs/secondary_interface/secondary_interface.md

```diff
@@ -0,0 +1,155 @@
+# Network Observability secondary interface support
+
+By: Mohamed S. Mahmoud
+
+Network observability for secondary interfaces with Multus and SR-IOV plugins
+in Kubernetes can be a complex task, but it's crucial for monitoring and
+troubleshooting network issues in a Kubernetes cluster.
+Here's an overview of how you can achieve network observability for secondary
```

Comment:
I'd add something here about how we've already added this ability with network observability operator, in OCP, with our 1.4 version

Reply:
u are referring to this section https://github.com/netobserv/documents/pull/56/files#diff-a12dd20328c486e3729e49cbcc9d666fb2371b876df01772d54dc35b2ae31b29R26